src/sentry/receivers/features.py

```diff
@@ -0,0 +1,231 @@
+from __future__ import absolute_import
+import six
+
+from sentry.adoption import manager
+from sentry.interfaces.base import get_interface
+from sentry.models import FeatureAdoption
+from sentry.plugins import IssueTrackingPlugin, IssueTrackingPlugin2
+from sentry.plugins.bases.notify import NotificationPlugin
+from sentry.receivers.rules import DEFAULT_RULE_LABEL, DEFAULT_RULE_DATA
+from sentry.signals import (
+    alert_rule_created,
+    event_processed,
+    first_event_received,
+    project_created,
+    member_joined,
+    plugin_enabled,
+    user_feedback_received,
+    api_called,
+    issue_assigned,
+    issue_resolved_in_release,
+    advanced_search,
+    save_search_created,
+    inbound_filter_toggled,
+    sso_enabled,
+    data_scrubber_enabled,
+)
+from sentry.utils.javascript import has_sourcemap
+
+DEFAULT_TAGS = frozenset(['level', 'logger', 'transaction', 'url', 'browser', 'sentry:user',
+    'os', 'server_name', 'device', 'os.name', 'browser.name', 'sentry:release', 'environment',
+    'device.family', 'site', 'version', 'interface_type', 'rake_task', 'runtime', 'runtime.name',
+    'type', 'php_version', 'app', 'app.device', 'locale', 'os_version', 'device_model',
+    'deviceModel', 'sentry_version'])
+
+
+# First Event
+@first_event_received.connect(weak=False)
+def record_first_event(project, group, **kwargs):
+    FeatureAdoption.objects.record(
+        organization_id=project.organization_id,
+        feature_slug="first_event",
+        complete=True)
+
+
+@event_processed.connect(weak=False)
+def record_event_processed(project, group, event, **kwargs):
+    # Platform
+    if group.platform in manager.location_slugs('language'):
+        FeatureAdoption.objects.record(
+            organization_id=project.organization_id,
+            feature_slug=group.platform,
+            complete=True)
+
+        # Frameworks
+        if event.data.get('sdk'):
+            if len(event.data.get('sdk').get('name').split(':')) == 2:
```

Comment:
I'm going to delete this for now. Just had a conversation with @dcramer and @bretthoerner that we'll likely want framework as a separate attribute, instead of using this colon syntax.